spec/views/granules/descriptor_document_spec.rb

```diff
@@ -153,22 +153,30 @@
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}&amp;clientId=foo\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}&amp;clientId=foo\">")
   end
   it "is possible to create a collection-specific granule open search descriptor document with a blank client id" do
     stub_client_id = stub_model(ClientId)
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
   end
   it "is possible to create a collection-specific granule open search descriptor document with a whitespace client id" do
     stub_client_id = stub_model(ClientId)
     stub_client_id.clientId = ' '
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
```

Comment:
do you mean to send nothing to q?

Reply:
If no dataset id is assigned (as per original test), yes.

Reply:
and the parameter is required I guess?